build/add-cluster-script.yaml

```diff
@@ -58,6 +58,25 @@ steps:
         - name: compare_versions
           on:
             - success
+      branch:
+        only: ["main"]
+
+  build_dev_image:
+    stage: Release
+    type: build
+    title: build and push add-cluster dev image
+    working_directory: manifests/add-cluster
+    image_name: codefresh/csdp-add-cluster
+    tag: ${{CF_BRANCH}}
+    registry: ${{REGISTRY_INTEGRATION_QUAY}}
+    buildkit: true
+    when:
+      steps:
+        - name: compare_versions
+          on:
+            - success
+      branch:
+        ignore: ["main"]
```

Comment:
pr pushes do not trigger the pipeline. but it can be used for a manual pipeline run, just to build the matching image